res/layout/view_profile_picture.xml

```diff
@@ -19,13 +19,9 @@
                 android:id="@+id/doubleModeImageView1"
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
+                android:alpha="0.65"
```

Comment:
I think I like this, but let's first implement the design we have and then think about changing it